pkg/apis/core/validation/validation.go

```diff
@@ -2663,10 +2663,8 @@ func validatePullPolicy(policy core.PullPolicy, fldPath *field.Path) field.Error
 	allErrors := field.ErrorList{}
 
 	switch policy {
-	case core.PullAlways, core.PullIfNotPresent, core.PullNever:
+	case core.PullAlways, core.PullIfNotPresent, core.PullNever, "":
```

Comment:
Does this make policy an optional field? Was that intentional?

Reply:
I misunderstood the order of container creation. In fact, this check happens after `SetDefaults_EphemeralContainer`